ui/src/main/resources/Meeting/Code/MeetingSheet.xml

```diff
@@ -205,7 +205,9 @@
       (((
         (% class="col-sm-12 col-xs-12" %)
         (((
-          #if($xcontext.action == 'view' &amp;&amp; $doc.getValue('displayMap') == 1 &amp;&amp; "${doc.getValue('place')}" != '')
+          #if($xcontext.action == 'view'
+              &amp;&amp; $doc.getValue('displayMap') == 1
+              &amp;&amp; "${doc.getValue('place')}" != '')
```

Comment:
If it's enough space in a line, I would prefer to use it :) There's no need to have new line for each condition.

Reply:
Changing it in a moment.